src/routes/report.tsx

```diff
@@ -16,11 +16,14 @@ const reportPageStyle = css`
   padding: 2rem;
   justify-content: center;
 `
+
+// defines qualities of the report area (white background) on the page
```

Comment:
In UI design we usually refer to something like this as a card. Tends to have white background, shadow, rounded corners, etc. That's what I was trying to describe with the name of the variable. Do you have a suggestion for a better variable name that we could use here?